### PR TITLE
Feature: breadcrumbs

### PR DESCRIPTION
**Pull request by wolfderechter**
## ID
 DGS-314

 ## Description

Subsidiedatabank is missing breadcrumbs to go back

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

Run frontend and verify working breadcrumbs

### CODE REVIEW

File: app/components/bread-crumb.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      crumbs: [{ label: 'Subsidiebeheer' }],
+    },
+    {
+      route: 'subsidy.detail.step.new',
+      crumbs: [{ label: 'Bekijk subsidieaanvraag' }],
+    },
+    {
+      route: 'subsidy.detail.step.detail',
+      crumbs: [{ label: 'Bekijk subsidieaanvraag' }],
+    },
```
Comment (Windvis) [inline app/components/bread-crumb.js:19]: Silly question, but how do you get to these routes without manually editing the url?

Comment (wolfderechter) [reply app/components/bread-crumb.js:19]: Try going to page 2 of the subsidies and you should be able to click through to the detail view. Not all subsidy types are available yet in the subsidiedatabank

**Review by Windvis — COMMENTED** [review]
Looks good to me,

The only strange thing I noticed is that there is already an "exit" link on the details page:

image

I think that can be removed if the breadcrumbs are added?

**Comment by wolfderechter** [comment]
> Looks good to me,
> 
> The only strange thing I noticed is that there is already an "exit" link on the details page:
> 
> image
> 
> I think that can be removed if the breadcrumbs are added?

Indeed, I was gonna replace it with the 'download pdf' button in a different PR, done

File: app/components/bread-crumb.js (hunk 1 above)
Comment (Windvis) [reply app/components/bread-crumb.js:19]: Wait, I thought you were going to remove the breadcrumb for the .new route? :smile:

Comment (wolfderechter) [reply app/components/bread-crumb.js:19]: Oops, done here https://github.com/lblod/frontend-subsidiedatabank/pull/26
too much context-switching :sweat_smile:

**Pull request merged**